src/Components/Web/src/SupplyParameterFromFormAttribute.cs

```diff
@@ -22,4 +22,7 @@ public sealed class SupplyParameterFromFormAttribute : CascadingParameterAttribu
     /// the form data and decide whether or not the value needs to be bound.
     /// </summary>
     public string? Handler { get; set; }
+
+    /// <inheritdoc />
+    protected internal override bool SingleDelivery => true;
```

Comment:
This can be `private protected` and we save ourselves the public API bit.

Reply:
We have to be able to read this value from inside other code in `M.A.Components` (e.g., in `CascadingParameterState`), so I think it has to allow `internal` access. Am I missing something?

Reply:
Actually I see we could also have a separate `internal` property that reads the `private protected` value. Interesting idea.

Reply:
Done